CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.34.0
+
+* `FEAT`: add promise polyfill for phantom-js ([`e9d5969f`](https://github.com/bpmn-io/bpmn-js-properties-panel/commit/e9d5969f141e1424a3026fc32f2a380e40507118))

```

Comment:
Is it a feature? It's only a change in the test infrastructure, so I'd go for `CHORE`

Reply:
alrighty